src/plugins/sentryPlugin.ts

```diff
@@ -38,7 +51,7 @@ export const sentryPlugin: ApolloServerPlugin = {
           // Only report internal server errors,
           // all errors extending ApolloError should be user-facing
           if (
-            errorCodes.includes(err.extensions?.code?.toString()) ||
+            NO_REPORT_ERRORS.has(err.extensions?.code?.toString()) ||
             err.originalError instanceof NotFoundError
```

Comment:
do we still need this line with the refactor?

Reply:
Good catch